test/base/test_mps.jl

```diff
@@ -755,6 +755,20 @@ end
     truncate!(M; site_range=3:7, maxdim=2)
     @test linkdims(M) == [2, 4, 2, 2, 2, 2, 8, 4, 2]
   end
+
+  @testset "truncate! with callback!" begin
+    nsites = 10
+    nbonds = nsites - 1
+    mps_ = basicRandomMPS(nsites; dim=10)
```

Comment:
```suggestion
    mps_ = random_mps(nsites; dim=10)
```
Was there a reason to use `basicRandomMPS`as opposed to `random_mps`? I don't remember what `basicRandomMPS` was defined for in the first place.